dojo-dungeon/todo-assets/Fehler_Syntax/FehlerhafteKlasse.java

```diff
@@ -1,33 +1,32 @@
-// what is wrong here?
+// import *;
```

Comment:
über den wert des kommentars hier kann man sich streiten. die leuten werden die datei in idea öffnen, und dann hast du die importe automatisch ....

Reply:
Ja ok, die Zeile könnte weg. Btw. ich hatte die Kommentarzeichen eingefügt, weil andernfalls das spotless / unsere ci durcheinanderbringt.